.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -37,6 +37,11 @@ defaults:
 ###################################################################
 on:
   workflow_call:
+    secrets:
+      registry-username:
```

Comment:
How about adding a note in the description that it should only be used for high-entropy value, and ask to read the doc (with a link).
We could also add it for the generic container.

Reply:
We have a note about this in the docs for the container generator.

See `registry-username` under 'Secrets:'
https://github.com/slsa-framework/slsa-github-generator/blob/6a2cc1cb559a81ffbbcd4248026c6ea89bdab2b6/internal/builders/container/README.md#workflow-inputs